### PR TITLE
Chrisjackson coil 1616 jest code cov

### PR DESCRIPTION
**Pull request by chris-jackson-actionqa**

### CODE REVIEW

File: packages/coil-anonymous-tokens/jest.config.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+  testMatch: ['<rootDir>/test/jest/**/*.test.[jt]s?(x)'],
+
+  // coverage
+  coverageReporters: ['json', 'lcov', 'text', 'clover', 'html'],
+  coveragePathIgnorePatterns: ['/node_modules/'],
+  coverageDirectory: 'results/coverage',
+  collectCoverageFrom: [
+    'src/**/*.{js,jsx,ts,tsx}',
+    // additional directories not under "src"
+    'assets/**/*.{js,jsx,ts,tsx}',
```
Comment (sublimator) [inline packages/coil-anonymous-tokens/jest.config.js:13]: Which package has the `assets` ?

File: jest.config.js [hunk 2]
```diff
@@ -37,6 +37,19 @@ let config = {
...
   }
+
+  // coverage
+  // coverageReporters: ['json', 'lcov', 'text', 'clover', 'html'],
+  // coverageDirectory: 'results/coverage',
+  // collectCoverageFrom: [
+  //   // every top level "src" directory under packages
+  //   'packages/coil-extension/src/**/*.{js,jsx,ts,tsx}',
+  //   // additional directories not under "src"
+  //   'packages/coil-extension/assets/**/*.{js,jsx,ts,tsx}',
```
Comment (sublimator) [inline jest.config.js:48]: image
Yeah, only seeing some illustrator file under coil-extension/assets

File: packages/coil-anonymous-tokens/jest.config.js (hunk 1 above)
Comment (chris-jackson-actionqa) [reply packages/coil-anonymous-tokens/jest.config.js:13]: coil-extension

File: jest.config.js (hunk 2 above)
Comment (chris-jackson-actionqa) [reply jest.config.js:48]: Weird.  I thought I saw some js files.

**Pull request closed without merging**